src/Microsoft.VisualStudio.Services.Agent/HostTraceListener.cs

```diff
@@ -65,6 +68,12 @@ public override void TraceEvent(TraceEventCache eventCache, string source, Trace
             WriteHeader(source, eventType, id);
             WriteLine(message);
             WriteFooter(eventCache);
+
+            if (!_diagErrorDetected && eventType < TraceEventType.Warning)
```

Comment:
So basically, we hope the first error we traced into the diag log will help the customer?

Reply:
the idea is that if we see an error, we tell the user on the console to look at the trace log

Reply:
ok